src/main/java/filter/expression/Qualifier.java

```diff
@@ -214,16 +192,16 @@ public static void setCurrentTime(LocalDateTime dateTime) {
     }
 
     public boolean isEmptyQualifier() {
-        return name.isEmpty() && content.isPresent() && content.get().isEmpty();
+        return type == QualifierType.EMPTY;
     }
 
     public boolean isFalseQualifier() {
-        return name.equalsIgnoreCase("false") && content.isPresent() && content.get().isEmpty();
+        return type == QualifierType.FALSE;
```

Comment:
Oh. What I meant was instead of using `type == QualifierType.FALSE` and `type == QualifierType.EMPTY`, we use `this == Qualifier.FALSE` and `this == Qualifier.EMPTY`. In case someone accidentally add a bug where the type is wrongly set as `FALSE` or `EMPTY` but it has content.